### PR TITLE
Job INIT kill bug, client bug, verbosity adjustments

### PR DESCRIPTION
**Pull request by mprimi**

### CODE REVIEW

File: genie-web/src/main/java/com/netflix/genie/web/jobs/workflow/impl/JobKickoffTask.java [hunk 1]
```diff
@@ -152,11 +151,6 @@ public void executeTask(@NotNull final Map<String, Object> context) throws Genie
                 .redirectOutput(new File(jobExecEnv.getJobWorkingDir() + JobConstants.GENIE_LOG_PATH))
                 .redirectError(new File(jobExecEnv.getJobWorkingDir() + JobConstants.GENIE_LOG_PATH));
 
-            //
-            // Check if file can be executed. This is to fix issue where execution of the run script fails because
```
Comment (tgianos) [inline genie-web/src/main/java/com/netflix/genie/web/jobs/workflow/impl/JobKickoffTask.java]: We're sure this isn't an issue still?

File: genie-client/src/main/java/com/netflix/genie/client/JobClient.java [hunk 2]
```diff
@@ -574,7 +574,7 @@ public JobStatus waitForCompletion(final String jobId, final long blockTimeout,
             if (System.currentTimeMillis() - startTime < blockTimeout) {
                 Thread.sleep(pollTime);
             } else {
-                throw new GenieTimeoutException("Timed out waiting for job to finish");
+                throw new GenieTimeoutException("Timed out waiting for job to finish: " + jobId);
```
Comment (tgianos) [inline genie-client/src/main/java/com/netflix/genie/client/JobClient.java]: This happens in the logs but the code is on the client side. Are you sure this is the error log message you think it is?

File: genie-web/src/main/java/com/netflix/genie/web/jobs/workflow/impl/JobKickoffTask.java (hunk 1 above)
Comment (mprimi) [reply genie-web/src/main/java/com/netflix/genie/web/jobs/workflow/impl/JobKickoffTask.java]: I really don't see how `touch` would have made any difference.

But i did deploy to my cluster and run the smoke tests.

File: genie-client/src/main/java/com/netflix/genie/client/JobClient.java (hunk 2 above)
Comment (mprimi) [reply genie-client/src/main/java/com/netflix/genie/client/JobClient.java]: Yes, smoke tests and gradle plugin test for example print this but no idea what the corresponding job is.

**Pull request merged**